build.gradle

```diff
@@ -15,10 +21,10 @@ sourceCompatibility = 11
 targetCompatibility = 11
 
 ext {
-    grpcVersion                 = '1.50.2'
-    protobufVersion             = '3.23.2'
-    serviceGeneratorVersion     = '3.3.1'
-    sharedDir           = file("${project.rootDir}/shared")
+    grpcVersion = '1.58.0'
```

Comment:
Please use 1.56.0 as bom or update dependencies version in bom to avoid vulnerability